amplify/backend/function/fetchDomainBalance/src/utils.js

```diff
@@ -26,44 +27,51 @@ const getActionWithFinalizedDate = (action) => ({
   finalizedDate: getActionFinalizedDate(action)
 })
 
-const getPeriodFromNow = (timeframePeriod, timeframeType) => {
+const getPeriodFromNow = (timeframePeriod, timeframeType, timeframePeriodEndDate) => {
```

Comment:
sorry, could you please explain why this is called "fromNow"? thank you ❤️

Reply:
And another thing, would it be better to refactor arguments?
like this 
```suggestion
const getPeriodFromNow = (timeframeType, timeframePeriod, timeframePeriodEndDate) => {
```
or
```suggestion
const getPeriodFromNow = ({ timeframeType, timeframePeriod, timeframePeriodEndDate }) => {
```